docs/content/documentation/getting-started/installation.md

```diff
@@ -49,11 +49,17 @@ $ sudo dpkg -i zola_<version>_amd64_debian_<debian_version>.deb
 
 ### Fedora
 
-Zola has been available in the official repositories since Fedora 29.
+Zola was included with Fedora 29 through Fedora 35.
 
 ```sh
 $ sudo dnf install zola
```

Comment:
Let's remove that completely. No need to point users to years old versions.

Reply:
Yep https://github.com/getzola/zola/issues/2078 it would still cause issues I think